### ISSUE TITLE
Crash while Generating Volcano and a river from Streams

### ISSUE THREAD

**Issue by canitzp** [Team-RTG/Realistic-Terrain-Generation#907]
I flied through my world and everything worked fine, but then my mc freeze and this crash happened:
http://pastebin.com/hbDztRBA
If you need the log file than say, cause it is to long for pastebin, but one line is maybe interesting:
[13:07:48] [Server thread/ERROR] [FML/]: Exception caught during firing event net.minecraftforge.event.terraingen.ChunkGeneratorEvent$ReplaceBiomeBlocks@6d9dab4e:

**Comment by whichonespink44** [comment]
Thanks for reporting this.

Would you be able to upload your log to [Gist](https://gist.github.com/) or [Dropbox](https://www.dropbox.com/) so we can look into it further?

**Comment by canitzp** [comment]
Here it is: http://canitzp.de/fml-client-latest.log

**Comment by Zygus42** [comment]
Volcano gen also crashes with other mods that do world gen.
[crash-2016-08-21_01.36.00-server.txt](https://github.com/Team-RTG/Realistic-Terrain-Generation/files/428692/crash-2016-08-21_01.36.00-server.txt)
[crash-2016-08-21_01.12.47-server.txt](https://github.com/Team-RTG/Realistic-Terrain-Generation/files/428693/crash-2016-08-21_01.12.47-server.txt)

**Comment by whichonespink44** [comment]
Would you guys be able to post your rtg.cfg file as well? The crash seems to suggest a custom volcano block that it doesn't like for some reason.

**Comment by canitzp** [comment]
Default, I haven't changed it

**Comment by whichonespink44** [comment]
Hmm, can you tell me what it has for the "Volcano mix 1 block ID" setting?

**Comment by canitzp** [comment]
BiomesOPlenty:ashStone

**Comment by whichonespink44** [comment]
Thanks, I know what the problem is - it will be fixed in the next snapshot release.

**Comment by whichonespink44** [comment]
Fixed in 1bafa89 and will be in the next release.

If you want to get around the crash in the meantime, just set all volcano block config options in rtg.cfg to their default vanilla values:

```
    # The main block to use for the surface of the volcano.
    #  [default: minecraft:obsidian]
    S:"Volcano block ID"=minecraft:obsidian

    # The meta value of the volcano block.
    #  [range: 0 ~ 15, default: 0]
    I:"Volcano block meta value"=0

    # The block ID of the 1st volcano mix block.
    #  [default: minecraft:cobblestone]
    S:"Volcano mix 1 block ID"=minecraft:cobblestone

    # The meta value of the 1st volcano mix block.
    #  [range: 0 ~ 15, default: 0]
    I:"Volcano mix 1 block meta value"=0

    # The block ID of the 2nd volcano mix block.
    #  [default: minecraft:gravel]
    S:"Volcano mix 2 block ID"=minecraft:gravel

    # The meta value of the 2nd volcano mix block.
    #  [range: 0 ~ 15, default: 0]
    I:"Volcano mix 2 block meta value"=0

    # The block ID of the 3rd volcano mix block.
    #  [default: minecraft:coal_block]
    S:"Volcano mix 3 block ID"=minecraft:coal_block

    # The meta value of the 3rd volcano mix block.
    #  [range: 0 ~ 15, default: 0]
    I:"Volcano mix 3 block meta value"=0
```
